src/NSwag.SwaggerGeneration/Processors/Collections/OperationProcessorCollection.cs

```diff
@@ -21,5 +21,23 @@ public T TryGet<T>()
         {
             return (T)this.FirstOrDefault(p => p is T);
         }
+
+        /// <summary>Replaces the first element of type <typeparamref name="T"/>
+        /// with <paramref name="newItem"/>.</summary>
+        /// <typeparam name="T">The operation processor type to replace.</typeparam>
+        /// <param name="newItem">The replacement item.</param>
+        /// <returns>true, if an item was replaced; otherwise false.</returns>
+        public bool Replace<T>(IOperationProcessor newItem) where T : IOperationProcessor
+        {
+            var item = this.OfType<T>().FirstOrDefault();
+
+            if (item != null)
+            {
+                this.SetItem(this.IndexOf(item), newItem);
```

Comment:
Remove this.

Reply:
Just to make sure I understand correctly:

You want me to remove the Replace-method from the collection and do the replacement of the OperationTagsProcessor with explicit code inside AspNetCoreToSwaggerGeneratorSettings instead?

Reply:
Sorry no, just remove the `this.` prefix :-)

Reply:
LOL... done.